eng/packaging.targets

```diff
@@ -1,4 +1,13 @@
 <Project InitialTargets="_OverridePackDependsOnForCsProjToPkgProj">
+  
+  <PropertyGroup>
+    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificContentInPackage);AddRefAssemblies;AddRuntimeSpecificAssemblies;LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
+    <TargetsForTfmSpecificDebugSymbolsInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRefDebugSymbolsWithTfm;AddRuntimeDebugSymbolsWithTfm</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <NoWarn Condition="'$(IsPackable)' == 'true'">$(NoWarn);NU5131</NoWarn>
```

Comment:
Can you add a comment that links to https://github.com/NuGet/Home/issues/8684?  Essentially it's impossible for a CSProj that uses `ref` to satisfy `NU5131` since CSProj-pack doesn't expose any mechanism to set the `<references>` in the nuspec.
```suggestion
    <!-- impossible to satisfy NU5131 for packages containing `ref` -- https://github.com/NuGet/Home/issues/8684 -->
    <NoWarn Condition="'$(IsPackable)' == 'true'">$(NoWarn);NU5131</NoWarn>
```